### PR TITLE
backup: update sse to string

### PR DESCRIPTION
**Pull request by DanielZhangQD**
SSE configures the server-side encryption algorithm used when storing this object in S3 and the type should be string.

### CODE REVIEW

File: proto/backup.proto [hunk 1]
```diff
@@ -118,7 +118,7 @@ message S3 {
     string prefix = 4;
     string storage_class = 5;
     // server side encryption
-    bool sse = 6;
+    string sse = 6;
```
Comment (kennytm) [inline proto/backup.proto]: @overvenus does this `6` need to be changed to another tag?

Comment (overvenus) [reply proto/backup.proto]: I think it's okay to keep `6` since the new `S3` is not used yet.  BTW, why does it need to be changed to `string`?

Comment (DanielZhangQD) [reply proto/backup.proto]: It is a `string` instead of `bool`, please check detail in the [here](https://rclone.org/s3/#s3-server-side-encryption).

**Comment by DanielZhangQD** [comment]
> LGTM. (Someone please send a PR to tikv master to update kvproto again after this is merged)

The kvproto for S3 is not updated in https://github.com/tikv/tikv/pull/6186, I think @yiwu-arbug should update the kvproto for S3.

**Comment by kennytm** [comment]
Yes, tikv/tikv#6186 was merged before this PR, so a separated PR is need to update kvproto on TiKV master again. No need to cherry-pick to TiKV 3.1 though, I've included this in tikv/tikv#6207 already.

**Pull request merged**